.github/workflows/generate_addressbooks.yaml

```diff
@@ -24,15 +24,13 @@ jobs:
           git clone https://github.com/balancer/balancer-deployments.git
           export DEPLOYMENTS_REPO_ROOT_URL=`pwd`/balancer-deployments
           pip3 install -r bal_addresses/requirements.txt
-          python3 gen_core_pools.py
-          python3 gen_pools_and_gauges.py
           python3 transform-deployments.py
```

Comment:
Transform deployments is the thing that generates addressbook.json, which is the thing that most people use.

It looks like you ru generate_addresses after each thing, but not transform_deployments.

Maybe in the end we need to just refactor gen_addresses and transform deployments.  For now it's a bit strange to be running one and not the other.  

Not sure if we should block this PR because of this, eveything will end up eventually consistent, but given the current setup it could take a couple weeks.

Reply:
hmm ok good point. i shuffled the workflows in 389d983 a bit:
- `generate_addressbooks.yaml`: create jsons for pools, gauges, chains, and the mono addressbook
- `generate_permissions.yaml`: create permissions jsons
- `generate_core_pools.yaml`: create core pools json

Reply:
OK but addressbook.json + all the slashbooks (reverse nad forward) per chain all need to be up to date.  It seems like transform_deployments generates the addressbook and gen_addresses generates the flatbooks, and then all these other things generate artifacts that these 2 include.  This seems like it  will get messy.  Let me look at it a bit more.